client/src/libs/components/modal/styles.module.scss

```diff
@@ -21,31 +21,31 @@
   align-items: center;
 }
 
-@media only screen and (max-width: 767px) {
+@media only screen and (width >= 767px) {
```

Comment:
It's not a good idea to make some changes which are not related to current task

Reply:
This depend to update stylelint-config-standard-scss that have new rules for this case.
I can`t commit changes if don`t fix stylelint errors(

Reply:
Ok, let's stay with it